care/emr/api/viewsets/report/template.py

```
@@ -0,0 +1,253 @@
+            from care.emr.reports.context_builder import types  # noqa
+            from care.emr.reports.context_builder.type_registry import FieldTypeRegistry
+
+            builder = ReportContextBuilder()
+            schema = builder.get_full_schema()
+
+            # Add registered type definitions
+            schema["types"] = FieldTypeRegistry.get_all()
+
+            return Response(schema)
+
+        except Exception as e:
```

Ruff (0.14.5)

Do not catch blind exception: `Exception`

(BLE001)